solr/bin/solr.cmd

```diff
@@ -1520,7 +1521,9 @@ for %%a in (%*) do (
    ) else (
       set "option!option!=%%a"
       if "!option!" equ "-d" set "SOLR_SERVER_DIR=%%a"
+      if "!option!" equ "--server-dir" set "SOLR_SERVER_DIR=%%a"
       if "!option!" equ "-s" set "SOLR_HOME=%%a"
+      if "!option!" equ "--solr-home" set "SOLR_HOME=%%a"      
       if not "!option!" equ "-s" if not "!option!" equ "-d" (
         set "AUTH_PARAMS=!AUTH_PARAMS! !option! %%a"
       )
```

Comment:
Is this code block still gonna work correctly if `--server-dir` and `--solr-home` is used instead?

I feel like we have to somehow get rid of these lists of options, they seem very error-prone and can be easily forgotten when we make changes.

Reply:
Yeah, good darn question.  I also think we have extra options parsing in the `.cmd` that doesn't exist in `bin/solr`...   I think we have places where we could be leaning more on the existing java parsing...

Reply:
I opened this: https://github.com/apache/solr/pull/2760